terraform-modules/github-runner/variables.tf

```diff
@@ -0,0 +1,73 @@
+variable "vault-role-id-path" {
+  type        = string
+  description = "A `gs://...` path that the instance's service account can read the vault role ID from"
+
+  validation {
+    condition     = length(regexall("^gs://.*", var.vault-role-id-path)) == 1
+    error_message = "The path to a vault role ID must be given as a `gs://...` path"
+  }
+}
+
+variable "vault-secret-id-path" {
+  type        = string
+  description = "A `gs://...` path that the instance's service account can read the vault secret ID from"
+
+  validation {
+    condition     = length(regexall("^gs://.*", var.vault-secret-id-path)) == 1
+    error_message = "The path to a vault secret ID must be given as a `gs://...` path"
+  }
+}
+
+variable "github-personal-access-token-path" {
+  type        = string
+  description = "A `secret/...` path within Vault to use to get a GH PAT to register the runner"
+
+  validation {
+    condition     = length(regexall("^secret/.*", var.vault-secret-id-path)) == 1
+    error_message = "The path to a Vault Secret ID must be given as a `gs://...` path"
+  }
+}
+
+variable "repo" {
+  type        = string
+  description = "The name of the GitHub repo to be a runner for, without the owner prefix"
```

Comment:
Even though we're just creating a runner for WFL, you can use the same runner for multiple repos right?

Reply:
Unfortunately no, runners are per-repository. This is intended for the gotc-deploy repo, we can only use these runners safely on private repositories.

Reply:
Ah okay, thanks for clarifying that!